src/picterra/tracer_client.py

```diff
@@ -256,12 +256,7 @@ def analyze_plots(
         _check_resp_is_ok(resp, "Couldn't start analysis")
         op_result = self._wait_until_operation_completes(resp.json())
         analysis_id = op_result["results"]["analysis_id"]
-        resp = self.sess.get(
-            self._full_url(f"plots_groups/{plots_group_id}/analysis/{analysis_id}/")
-        )
-        _check_resp_is_ok(resp, f"Failure to get analysis {analysis_id}")
-        analysis_data = resp.json()
-        return analysis_data
+        return analysis_id
```

Comment:
This is technically breaking, but since we haven't stabilized the Tracer API yet and are planning to release a first version officially supporting it soon, we're fine.

(just thinking out loud)

Reply:
Yes, I also feel we could go for a "clean" way and bump version to 3.0.0

Reply:
Would make sense to me, let's do that.